### ISSUE TITLE
Backcountry Navigator and aprsdroid

### ISSUE THREAD

**Issue by Blastco2** [ge0rg/aprsdroid#162]
Edit:   It is possible to see recieved locations on Backcountry navigator topo map.  On the station info screen, press menu, then, external map source......  works great!  Many thanks for this feature!

Myself and friends spend a fair amount of time in the backcountry of the Rocky Mountains.  We use a topo mapping app on our android devices called Backcountry Navigator.  Like Aprsdroid it does not need cell sevice to work.  There is an ongoing feature request with Crittermap software to have waypoint information from Aprsdroid to be displayed on BCN.  I am wondering if this is feasible or would it be possible to have a topo map source for Aprsdroid.

Thanks
Tom Ryan
Menan, Id
KG7OTL
WQSR556
My backyard is ground zero for Eclipse 2017.

**Comment by ge0rg** [comment]
The offline mapping support in APRSdroid is a little limited due to the Mapsforge library, and I fear I won't be able to integrate other mapping systems. There is already a live position reporting API in APRSdroid that can be hooked into by other applications, and I could imagine providing database access to other apps, but that would require work on the other side developers.

**Comment by Blastco2** [comment]
Right after I posted the request, I found the external map feature/function
in aprsdroid.  It will load BCn and give the choices to center on the map,
create waypoint or goto..... Works very slick!   My use for this is
snowmobiling on the US National Forest.  We can go anywhere and it is
common for someone to become separated from the group then get stuck and
need help to get moving again.  With the new changes coming from the FCC,
we will be able to use Aprsdroid on the gmrs channels.  My non-ham friends
will be included.
Thanks for your reply and for building Aprsdroid!
Tom
KG7OTL
WQSR556
